.github/workflows/workflow.yml

```
@@ -184,8 +184,8 @@ jobs:
     needs: [ check-for-functional-changes ]
     if: needs.check-for-functional-changes.outputs.status == 'success'
     env:
-      PYPI_USERNAME: openfisca-bot
-      PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      PYPI_USERNAME: __token__
+      PYPI_TOKEN: ${{ secrets.PYPI_TOKEN_OPENFISCA_CORE }}
```

@Allan-CodeWorks, here is the token name: 
```suggestion
      PYPI_TOKEN: ${{ secrets.PYPI_TOKEN_OPENFISCA_BOT }}
```

---

Should work according to me!